pkg/storage/sqlstorage/collect.go

```diff
@@ -0,0 +1,92 @@
+package sqlstorage
+
+import (
+	"context"
+	"database/sql"
+	"embed"
+	"io/fs"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+//go:embed migrates
+var MigrationsFS embed.FS
+
+type HandlersByEngine map[string][]MigrationFunc
```

Comment:
you define this here but it's not used in this file ? maybe we can define it in migrate.go ?